.github/workflows/build.yaml

```
@@ -251,9 +251,58 @@ jobs:
           path: |
             image/rhel/docs
 
+  check-collector-images-exist:
+    # This job ensures that COLLECTOR_VERSION cannot be updated to a version for which
+    # the image was not successfully built on Konflux.
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag_suffix: ["", "-fast", "-fast-slim"]
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4
+
+    - name: Get COLLECTOR_VERSION
+      id: collector-version
+      run: |
+        echo "collector-version=$(make --quiet --no-print-directory collector-tag)" >> "${GITHUB_OUTPUT}"
+
+    - name: Check image exists
+      uses: stackrox/actions/release/wait-for-image@v1
+      with:
+        token: ${{ secrets.QUAY_RHACS_ENG_BEARER_TOKEN }}
+        image: rhacs-eng/collector:${{ steps.collector-version.outputs.collector-version }}${{ matrix.tag_suffix }}
+        limit: 300
+
+  check-scanner-images-exist:
+    # This job ensures that SCANNER_VERSION cannot be updated to a version for which
+    # the image was not successfully built on GHA and Konflux.
```

There's one to remove here.

```suggestion
    # the image was not successfully built on Konflux.
```